proto/schedule/schedule.proto

```diff
@@ -0,0 +1,19 @@
+syntax = "proto3";
+
+package io.spinnaker.proto.schedule;
+
+option go_package = "github.com/spinnaker/rotation-scheduler/proto/schedule";
+
+message Shift {
+  // Must be in the format "Mon 02 Jan 2006".
+  string start_date = 1;
```

Comment:
A fair amount of the scheduling logic involves parsing this date and validating that it's actually a date---would there be a benefit to making this a `Timestamp` in the proto?  I guess we don't need any resolution beyond a day, but if I'm understanding the logic properly, I think that right now I could still type a string with minutes/seconds and the parsing would ignore the part at resolution smaller than a day.

Reply:
I originally went down this path, but a proto `Timestamp` serializes to a pair of integers - [seconds and nanos](https://github.com/protocolbuffers/protobuf/blob/master/src/google/protobuf/timestamp.proto#L127) - or effectively a non human readable format.

Reply:
Oh, I see, yeah I guess having this human-readable makes sense here.